src/quo2/components/graph/wallet_graph/utils.cljs

```diff
@@ -0,0 +1,29 @@
+(ns quo2.components.graph.wallet-graph.utils)
+
+(defn find-highest-value
+  [maps]
+  (reduce (fn [max-value map]
```

Comment:
Can we de-structure `map` to get `value`?
```suggestion
  (reduce (fn [max-value {:keys [value]}]
```

Reply:
Modified the whole function so it does not apply anymore